pages/index.vue

```diff
@@ -72,6 +72,47 @@ export default {
       heroButtonLink: '',
       heroButtonLabel: ''
     }
+  },
+
+  head() {
+    return {
+      title: 'SPARC Portal',
+      titleTemplate: '%s - SPARC Portal',
+      meta: [
+        {
+          name: 'description',
+          content: 'Stimulating Peripheral Activity to Relieve Conditions (SPARC)'
+        },
+        {
+          name: 'og:type',
+          content: 'website'
+        },
+        {
+          name: 'og:title',
+          content: 'SPARC Portal'
+        },
+        {
+          name: 'og:description',
+          content: 'Stimulating Peripheral Activity to Relieve Conditions (SPARC)'
+        },
+        {
+          name: 'og:site_name',
+          content: 'SPARC Portal'
+        },
+        {
+          name: 'twitter:card',
+          content: 'summary'
+        },
+        {
+          name: 'twitter:site',
+          content: '@sparc'
+        },
+        {
+          name: 'twitter:description',
+          content: 'Stimulating Peripheral Activity to Relieve Conditions (SPARC)'
```

Comment:
here

Reply:
summary of dataset:
example:
<meta data-vue-meta="1" name="twitter:description" content="Images collected from serial cryostat sectioning of a cryopreserved heart was used to reconstruct the 3D context. Transcriptional profiles taken from isolated single neurons and mapped back into the previously generated 3D context.">